index.js

```diff
@@ -15,3 +15,107 @@ function total() {
 
   return t
 }
+
+
+function getCart () {
+  return cart;
+}
+
+function addToCart (item) {
```

Comment:
Hi,
this is how I solved this function: 

function addToCart(item) {
  let price = Math.floor(Math.random() * 100);
  
   var newCart = {
     itemName: item,
     itemPrice: price
   }
   
   cart.push(newCart);
   return `${cart[cart.length - 1].itemName} has been added to your cart.`;
}